### PR TITLE
[15.0][MIG] fieldservice_vehicle

### PR DESCRIPTION
**Pull request by Vandan-OSI**

### CODE REVIEW

File: fieldservice_vehicle/README.rst [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Ffield--service-lightgray.png?logo=github
+    :target: https://github.com/OCA/field-service/tree/14.0/fieldservice_vehicle
```
Comment (Murtaza-OSI) [inline fieldservice_vehicle/README.rst]: Used oca-gen-addon-readme

File: fieldservice_vehicle/models/fsm_order.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        return self.person_id.vehicle_id.id or False
+
+    vehicle_id = fields.Many2one(
+        "fsm.vehicle", string="Vehicle", default=_get_default_vehicle
+    )
+
+    @api.onchange("person_id")
+    def _onchange_person_id(self):
+        if self.person_id.vehicle_id:
+            self.vehicle_id = self.person_id.vehicle_id.id
```
Comment (Murtaza-OSI) [inline fieldservice_vehicle/models/fsm_order.py]: Convert to single line code

**Review by murtuzasaleh — APPROVED** [review]

**Review by Murtaza-OSI — APPROVED** [review]

**Review by nikul-serpentcs — APPROVED** [review]
Code LGTM :+1: 

@max3903 @brian10048 can you please take a look?

**Review by Freni-OSI — APPROVED** [review]

**Pull request merged**